src/Core/Resolvers/DWSqlQueryBuilder.cs

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Data.Common;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Azure.DataApiBuilder.Core.Models;
+using Microsoft.Data.SqlClient;
+
+namespace Azure.DataApiBuilder.Core.Resolvers
+{
+    /// <summary>
+    /// Class for building DwSql queries.
+    /// </summary>
+    public class DWSqlQueryBuilder : BaseSqlQueryBuilder, IQueryBuilder
```

Comment:
NIT: might be useful to stick to camelcase convention i.e. DwSql along the lines of MsSql

Reply:
Fixed.